data/remnant/remnant outfits.txt

```diff
@@ -476,12 +477,22 @@ outfit "Anvil-Class Engine"
 	"thrust" 13.5
 	"thrusting energy" 2.05
 	"thrusting heat" 1.95
-	"turn" 336.0
+	"turn" 369.6
```

Comment:
I suggested the increased turn speed as an alternative to the slowing resistance, so if you aren't going to budge on the slowing resistance then revert the turn speed changes on the engines.

Reply:
Right now, there are two changes to Remnant steering. The buff to turning speed, and the addition of slowing resistance. As things stand right now, the buff to turning is a solid performance increase; while the slowing resistance is basically just flavor. 

One of the big reasons for having that slowing is to hint at the fact that the Remnant are preppers. As in, those survivalists who prep for even unlikely things. 
If the Remnant steering honestly could use a buff, then the buff to the steering should go through on its own merits.
As a separate issue, whether or not they have any slowing resistance should basically be a question of "do we want them to have this flavor or not." It is not, after all, a powerful or particularly useful thing, right now. But even a tiny amount of slowing resistance does mean that they have something that no other family of engines has. 

According to the math, a single inhibitor canon does 230.8 slowing damage per second, and the current smelter steering will dissipate 1.2 slowing per second. Which suggests to me that the slowing resistance there might have some benefit in helping a ship recover after it has gotten out of the line of fire, but probably won't even be noticeable while under the effect of it. Which kind of reinforces the fact that it is basically a lore / flavor thing, not a balance thing. Meanwhile, the Heliarch Repulsor does 600 slowing per second, and the Detainer does 4000 slowing damage per second. So the Steering's slowing resistance is even more "pure flavor" against them.